docs/react-v9/contributing/rfcs/react-components/convergence/z-index-handling.md

```diff
@@ -0,0 +1,319 @@
+# RFC: z-index handling
+
+---
+
+[@marcosmoura](https://github.com/marcosmoura)
+
+- [RFC: z-index handling](#rfc-z-index-handling)
```

Comment:
Love the TOC, FYI you can use [doctoc](https://github.com/thlorenz/doctoc) to autogenerate it 😉

Reply:
I was using the vscode extension that does this automatically, but nice suggestion!